src/main/java/com/mide/gangsaeng/board/BoardService.java

```diff
@@ -2,8 +2,12 @@
 
 import java.util.List;
 
+import com.mide.gangsaeng.common.CursorBasedRequest;
+import com.mide.gangsaeng.common.CursorBasedResponse;
+
 public interface BoardService {
     void write(BoardRequest request);
     BoardResponse read(long id);
     List<Board> getPage(int offset, int size);
+    CursorBasedResponse<Board> getPageByCursor(CursorBasedRequest request, int size);
```

Comment:
no need to fix
It's okay for now, but in this code, service layer should know about the detail of controller's response `CursorBasedResponse`. This is not good design but it's okay for toy project.